### ISSUE TITLE
Remove use local server settings? 

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-gui#1682]
I found the use local server settings confusing for multiple reason:

* For some node we have this settings but I'm not even sure that  it's change something (Docker for example) because we will always choose the server using the wizard
* If the server is remote use local server: mean run on controller  (See https://github.com/GNS3/gns3-gui/issues/1674)
* It's a settings lost alone in preference with no explanation of his purpose

Now we can create template for VPCS, switch... We have an alternative if you don't want to have the choice of local server.

**Comment by grossmj** [comment]
I agree this is confusing. Kinda legacy from the previous architecture.

This setting was used in the wizards, when checked we used to skip the first page of a wizard that asks on what server to run the node. The goal was not to confuse users that only run things locally with the concept of remote servers.

Currently, it is used to grey out some options only available locally. For instance if you take VPCS you can choose the vpcs executable path but only if you have checked the local server checkbox.

**Comment by noplay** [comment]
For the wizard we skip the first page if you don't have remote server.

For the settings we grey we can grey them if local doesn't support the technology (like IOU on Windows).

**Comment by grossmj** [comment]
I am ok to remove that setting but I do not want users who configure local paths (or other local settings, see memory optimization settings for IOS) think it is what is going to be used on their remote server or VM.
